pkg/provisioner/ironic/ironic.go

```diff
@@ -50,6 +59,15 @@ const (
 func init() {
 	// NOTE(dhellmann): Use Fprintf() to report errors instead of
 	// logging, because logging is not configured yet in init().
+	authStrategy = os.Getenv("IRONIC_AUTH_STRATEGY")
+	if authStrategy == "" {
+		fmt.Fprintf(os.Stderr, "Cannot start: No IRONIC_AUTH_STRATEGY variable set\n")
+		os.Exit(1)
+	}
+	if authStrategy != "noauth" && authStrategy != "http_basic" {
```

Comment:
Minor style thing, this is an obvious candidate for a switch statement:

```
switch authStrategy {
    case "noauth":
    case "http_basic":
        // Read other env vars
    case "":
        fmt.Fprintf(...)
        os.Exit(1)
    default:
        fmt.Fprintf(...)
        os.Exit(1)
}
```

Reply:
Sure I will update! I'm just wondering the messages I should use for  `case  ""` and `default` =)